js/src/components/paid-ads/continue-button.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+
+/**
+ * Renders Continue button on paid ad campaign create and edit page.
+ *
+ * @param {Object} props Props
+ * @param {Object} props.formProps Form props forwarded from `Form` component.
+ * @param {Function} props.onClick Function to handle the continue button click.
+ * @return {import(".~/components/app-button").default} The button.
```

Comment:
Related to https://github.com/woocommerce/google-listings-and-ads/pull/2577#discussion_r1792721347. The return type is not the `AppButton` function either. The return type is `JSX.Element` or the `@return` tag can be omitted.

Reply:
I have changed it to JSX.Element now. 

Return type was a bit confusing for me here. I was referring to `help-icon-button.js` which had the element import statement as return type. Some places even has `@return {AppButton}` as return type while other have JSX.Element.